docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
-The invite token: (=presets.tokens.first=)
-This token will expire in 60 minutes.
-
-This token enables Desktop Access.  See https://goteleport.com/docs/desktop-access/
-for detailed information on configuring Teleport Desktop Access with this token.
-```
-
-Copy the token to the Linux host where you will run the Desktop service as `/tmp/token`.
-
-(!docs/pages/includes/install-linux-enterprise.mdx!)
-
-Create `/etc/teleport.yaml` and configure it for desktop access. Update the `proxy_server`
-value to your Teleport proxy service or cloud tenant, and put the Windows machine address
-under `non_ad_hosts`:
-
-```yml
-version: v3
-teleport:
-  nodename: windows.teleport.example.com
-  proxy_server: teleport-proxy.example.com:443
-  auth_token: /tmp/token
-windows_desktop_service:
-  enabled: yes
-  non_ad_hosts:
-    - 192.0.2.156
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no
-```
-
-Note that without Active Directory, Teleport cannot automatically discover your
-Desktops. Instead you must define the Windows systems configured for access through
-Teleport in your config file, or use Teleport's [API](../api/introduction.mdx)
-to build your own integration. An example API integration is available on
-[GitHub](https://github.com/gravitational/teleport/tree/master/examples/desktop-registration).
-
-<Details title="Add labels to hosts">
-You can attach labels to your Windows hosts by matching to their hostnames.
-For example, to add the `cloud: ec2` label to hosts with EC2 private IP DNS names:
-
-```diff
-version: v3
-teleport:
-  nodename: windows.teleport.example.com
-  proxy_server: teleport-proxy.example.com:443
-windows_desktop_service:
-  enabled: yes
-  non_ad_hosts:
--    - 192.0.2.156
-+    - ip-192-0-2-156.us-east-2.compute.internal
-+ host_labels:
-+    - match: '.*\.us-east-2.compute.internal'
-+      labels:
-+        cloud: ec2
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no
+$ teleport-windows-auth-setup.exe install --cert=teleport.cer -r
 ```
 
-</Details>
-
-(!docs/pages/includes/start-teleport.mdx service="the Teleport Desktop Service"!)
-
-## Step 3/4. Configure Windows access
-
-In order to gain access to a remote desktop, a Teleport user needs to have the
+## Step 2/4. Configure the Windows Desktop Service
+
+Now that you have prepared the Windows computer to enroll in the cluster, the next steps
+involve:
+
+- Generating an invitation token on your administrative workstation.
+- Configuring a Linux server to run the Windows Desktop Service.
+
+To configure the Windows Desktop Service:
+
+1. Sign in to the Teleport cluster from your administrative workstation to authenticate your identity.
+
+1. Generate a short-lived invitation token by running the following command:
+   
+   ```code
+   $ tctl tokens add --type=windowsdesktop
+   ```
+   
+   The command displays information similar to the following:
+
+   ```code
```

Comment:
```suggestion
   ```text
```
We don't need the `code` label here since there aren't any command lines to copy individually. `text` has better contrast.